sqlair.go

```diff
@@ -81,7 +85,7 @@ func (db *DB) PlainDB() *sql.DB {
 	return db.sqldb
 }
 
-// Query holds the results of a database query.
+// Query represents a query on a database. It is designed to be run once.
```

Comment:
Can you only run it once though? I would assume you could pass in the values, store the query, and run it as much times as you want (think a select statement).

Reply:
You can run it multiple times, but it should only be run once, mainly because we store the context in the query so it should be used straight away. It being designed to be run once was something Gustavo and I decided ages ago.

Reply:
Right, then keep it as is.